src/app/machines/views/MachineList/MachineListTable/MachineListTable.tsx

```diff
@@ -49,6 +49,7 @@ import { actions as zoneActions } from "app/store/zone";
 export enum Label {
   Loading = "Loading machines",
   Machines = "Machines",
+  NoResults = "No machines match the search criteria.",
```

Comment:
I'm of the opinion it's helpful to make the name reflect the actual value (e.g. `NoMachines` instead of `NoResults`) where that makes sense.

This - arguably - makes for better readability as it doesn't require you to jump through sections of the code as much as you're reading it to understand what will be displayed.

What are your thoughts on this @huwshimi ?